coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/PlaybackToSecondary.scala

```diff
@@ -655,7 +655,8 @@ class PlaybackToSecondary[CT, CV](u: PlaybackToSecondary.SuperUniverse[CT, CV],
         return actions
       } catch {
         case e: Throwable =>
-          logger.info("Rolling back to end transaction due to thrown exception: {}", e.getMessage)
+          val msg = Option(e.getMessage).getOrElse(e.toString)
+          logger.info("Rolling back to end transaction due to thrown exception: {}", msg)
```

Comment:
```scala
        case e: Throwable =>
          logger.info("Rolling back to end transaction due to thrown exception:", e)
```
...will get you a stack trace in addition to the message, if you want that.

Reply:
just to make sure we're on the same page: you think it's okay to log e for all services going through data-coordinator?